.github/workflows/main.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: Build
+      id: build
+      uses: convox/actions/build@multipart
+      env:
+        CONVOX_PASSWORD: ${{ secrets.CONVOX_DEPLOY_KEY}}
```

Comment:
this should be a `with` argument of `password`

Reply:
If I do this it will be an input variable and Github will rename the env variable to `INPUT_CONVOX_PASSWORD`  which will break things unless I modify the entrypoint scripts to set an env variable called `CONVOX_PASSWORD` based on the value of `INPUT_CONVOX_PASSWORD` which I am happy to do but seems like maybe some unnecessary gymnastics.

Reply:
Let's do the gymnastics in the action so the users don't have to think about env vs with when using it.